notebooks/run_model_in_notebook.ipynb

```diff
@@ -46,7 +46,7 @@
     "# path to config file\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        input_dir, output_dir, config_path = PyStemmusScope.create_io_dir(nc_file, config_template)
</pre><p><code>Replace it with iostreamer.create_io_dir</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/STEMMUS_SCOPE_Processing/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>